#### Metatranscriptomic Monitoring of Mosquitoes
Infections by mosquito-borne pathogens account for nearly 500 million deaths a year, millions in reduction of diability-adjusted life years, and large scale die-offs of econonomically important domesticated animals. Studies have pointed at worldwide urbanization, warming, and mosquito transport as evidence of an increased probability of mosquito habitat expansion and mosquito-borne diseases in the next decade. It has also been shown in recent years, that mosquitoes harbor a high diversity of known and unknown viruses. Most are insect-specific, highly prevelent, belong to viral taxon that also contain mosquito-transmitted human pathogens, and might influence the ability of their hosts to transmit important arboviruses. We've initiated this project, in collaboration with the San Diego County Vector Control, to perform metatranscriptomic surveillance of mosquito pools in order to identify indicators of mosquito movements, transmission of human pathogens, novel viruses, and blood meal sources.

We perform metranscriptomic analysis using a standard Nextera protocol. We extract RNA from mosquito pools using the MagMAX Viral DNA/RNA Isolation Kit. We then deplete mosquito rRNA using a reverse-transcription/RNase H based protocol developed by [Fauver et al. 2019](https://doi.org/10.1016/j.virol.2018.12.020). Depleted RNA is used to create sequencing libraries using the Illumina Nextera protocol and sequenced on the Illumina MiSeq or NextSeq platform. An in-house pipeline is then used to analyze the raw sequencing reads. An overview of the entire library prep and analytical pipeline is shown below:

![Mosquito Monitoring Pipeline](/images/pipeline.png) 

Raw sequencing reads and contig classification can be found on [Google Cloud](https://console.cloud.google.com/storage/browser/andersen-lab_project_mosquito-metagenomics). The library preperation protocol, computational pipeline, visualizations, reports, and sample metadata will be hosted in this repository. 

**Disclaimer**. Please note that this data is still based on work in progress and should be considered preliminary. If you intend to include any of these data in publications, please let us know – otherwise please feel free to download and use without restrictions. We have shared this data with the hope that people will download and use it, as well as scrutinize it so we can improve our methods and analyses. Please contact us if you have any questions or comments – we’ll buy beers for #ResearchParasites that spot flaws and faults in the data and come up with improvements!
